packages/esm-patient-labs-app/src/test-results/grouped-timeline/grouped-timeline.tsx

```diff
@@ -216,7 +215,7 @@ const TimelineDataGroup = ({ parent, subRows, xScroll, setXScroll, panelName, se
     }
   }, [handleScroll]);
 
-  const onIntersect = (entries, observer) => {
+  const onIntersect = (entries) => {
```

Comment:
What on earth. This block does nothing. Time for `git blame`

Reply:
Oh, @vasharma05 two years ago. He was new then.

Please delete this whole block.